count_blocklist.go

```diff
@@ -0,0 +1,66 @@
+package p2p
+
+import (
+	"time"
+
+	"github.com/iotexproject/go-pkgs/cache"
+	"github.com/libp2p/go-libp2p-core"
+)
+
+// CountBlocklist is a count-based blacklist implementation. Entry is being added
+// to an LRU cache, once the count reaches a threshold the entry is blocked for a
+// certain timeout. The entry is then unblocked once the timeout expires
+type CountBlocklist struct {
+	count   int           // number of times an entry is added before being blocked
```

Comment:
tolerants?

Reply:
i think it's OK? above comments has explained how `CountBlocklist` works